### PR TITLE
[FIX] runbot_travis2docker: Change the updated_weblate whith the api

### PR DESCRIPTION
**Pull request by JesusZapata**
Change the field `updated_weblate` for calculated the difference directly with the repository of git

### CODE REVIEW

**Comment by JesusZapata** [comment]
@moylop260 
In this PR I deleted the field `updated_weblate` to calculated the difference directly with the repository of git and add one button `Force weblate` on the `runbot.branch`

image

File: runbot_travis2docker/models/runbot_branch.py [hunk 1]
```diff
@@ -76,35 +71,43 @@ def cron_weblate(self):
...
-                        updated_branch = component['branch']
+                    self._create_build(branch)
+                    updated_branch = component['branch']
+
+    @api.multi
+    def force_weblate(self):
+        for record in self:
+            self._create_build(record)
+
+    def _create_build(self, branch):
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_branch.py]: _create_wl_build

Comment (JesusZapata) [reply runbot_travis2docker/models/runbot_branch.py]: @moylop260 
Done, Thanks

File: runbot_travis2docker/models/runbot_branch.py [hunk 2]
```diff
@@ -76,35 +71,44 @@ def cron_weblate(self):
...
-                            'name': component['branch'],
-                            'uses_weblate': True})
-                        updated_branch = component['branch']
+                    self._create_build(branch)
+                    updated_branch = component['branch']
+
+    @api.multi
+    def force_weblate(self):
+        for record in self:
+            self._create_build(record)
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_branch.py:105]: _create_wl_build?

Comment (moylop260) [reply runbot_travis2docker/models/runbot_branch.py:105]: better, Why we have 2 methods
What about use just one?

File: runbot_travis2docker/models/runbot_branch.py [hunk 3]
```diff
@@ -76,35 +71,44 @@ def cron_weblate(self):
...
+                    if has_build:
                         continue
-                    change = None
-                    for record in changes['results']:
-                        if record['action'] == 17:
-                            change = record
-                            break
-                    if not change:
+                    remote = 'wl-%s' % project['slug']
+                    url_repo = (branch.repo_id.weblate_url.replace('api',
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_branch.py:82]: Use `'/'.join([...])`

File: runbot_travis2docker/models/runbot_branch.py [hunk 4]
```diff
@@ -76,35 +71,44 @@ def cron_weblate(self):
...
+                                component['slug'])
+                    try:
+                        subprocess.check_output(cmd + ['remote', 'add', remote,
+                                                url_repo])
+                    except subprocess.CalledProcessError:
+                        pass
+                    subprocess.check_output(cmd + ['fetch', remote])
+                    diff = subprocess.check_output(
+                        cmd + ['diff',
+                               '%(branch)s..%(remote)s/%(branch)s'
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_branch.py:94]: Use `origin/` for the default remote

Comment (JesusZapata) [reply runbot_travis2docker/models/runbot_branch.py:94]: @moylop260 
I try `git diff origin/10.0..wl-github_com_JesusZapata_openacademy-project_10_0_/10.0` but this show me on error `fatal: ambiguous argument`

image

**Pull request merged**